stacks/spica/src/bucket/pages/add/add.component.ts

```diff
@@ -18,6 +18,9 @@ export class AddComponent implements OnInit {
   bucketId: string;
   data: BucketRow = {};
   now: BucketRow;
+  minScheduleDate: Date = new Date();
+  scheduleDate: Date = new Date();
```

Comment:
We may use the `undefined` state of "scheduledDate" property to indicate whether a schedule is set? In this case we get rid of scheduleOn property?